src/app/containers/redirect/redirect.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { CookieService } from 'ngx-cookie';
+import { AppUtils } from 'src/app/utils/app-utils';
+
+@Component({
+    selector: 'app-redirect',
+    template: ``
+  })
+
+export class RedirectComponent implements OnInit {
+    constructor(private readonly cookieService: CookieService, private router: Router) {}
```

Comment:
Indentation looks off for a fair bit of this class... and certainly on this line.

Reply:
Agree, fixed it now. Thanks